### PR TITLE
making the rule position in PAN-OS create/edit rule non mandatory

### PR DESCRIPTION
**Pull request by Arsenikr**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/20647

## Description
making the playbooks more flexible by making Bottom the default rule position

### CODE REVIEW

File: Playbooks/playbook-PAN-OS_-_Create_Or_Edit_Rule.yml [hunk 1]
```diff
@@ -494,18 +500,20 @@ inputs:
...
-  required: true
+  required: false
   description: |-
     The position of the rule in the ruleset. Valid values are:
       * Top
       * Bottom
       * Before
       * After
+
+    The default is 'Bottom'
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_-_Create_Or_Edit_Rule.yml]: @yaron-libman

File: Playbooks/playbook-PAN-OS_-_Create_Or_Edit_Rule.yml [hunk 2]
```diff
@@ -494,18 +500,20 @@ inputs:
...
 - key: relative-rule-name
   value: {}
   required: false
-  description:  'If the rule-position that is chosen is before or after,
-    specify the rule name to which it is related.'
+  description: If the rule-position that is chosen is before or after, specify the
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_-_Create_Or_Edit_Rule.yml]: @yaron-libman

File: Playbooks/playbook-PAN-OS_-_Create_Or_Edit_Rule_CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Rule position is not mandatory anymore, the default is Bottom
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_-_Create_Or_Edit_Rule_CHANGELOG.md]: @yaron-libman

File: Playbooks/playbook-PAN-OS_DAG_Configuration.yml [hunk 4]
```diff
@@ -436,68 +442,61 @@ view: |-
...
+  value: {}
+  required: false
   description: |-
     The position of the rule in the ruleset. Valid values are:
       * Top
       * Bottom
       * Before
       * After
+
+    The default position is 'Top'
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_DAG_Configuration.yml]: @yaron-libman

File: Playbooks/playbook-PAN-OS_DAG_Configuration_CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Rule position is not mandatory anymore, the default is Top
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_DAG_Configuration_CHANGELOG.md]: @yaron-libman

File: Playbooks/playbook-PAN-OS_EDL_Setup.yml [hunk 6]
```diff
@@ -2,9 +2,9 @@ id: PAN-OS EDL Setup
 version: -1
 name: PAN-OS EDL Setup
 fromversion: 4.0.0
-description: Sets up external dynamic list in PAN-OS.\nin case file exists in web
-  server it will sync it to demisto. then it will create an EDL object and a matching
-  rule.
+description: "Sets up external dynamic list in PAN-OS.\nIn case file exists in web
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_EDL_Setup.yml]: @yaron-libman

File: Playbooks/playbook-PAN-OS_EDL_Setup.yml [hunk 7]
```diff
@@ -572,15 +579,16 @@ tasks:
...
+    id: "34"
+    taskid: b8a806b3-8153-4bcc-8a57-3446261f0ed9
     type: playbook
     task:
-      id: 0a29b0bd-db3c-428c-8892-8441eab72736
+      id: b8a806b3-8153-4bcc-8a57-3446261f0ed9
       version: -1
       name: PAN-OS - Create Or Edit Rule
-      description: ""
+      description: Creates or edits a Panorama rule and moves it into the desired
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_EDL_Setup.yml]: @yaron-libman

File: Playbooks/playbook-PAN-OS_EDL_Setup.yml [hunk 8]
```diff
@@ -684,19 +698,20 @@ inputs:
...
-  required: true
+  required: false
   description: |-
     The position of the rule in the ruleset. Valid values are:
       * Top
       * Bottom
       * Before
       * After
+
+    The default position will be 'Top'
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_EDL_Setup.yml]: @yaron-libman

File: Playbooks/playbook-PAN-OS_EDL_Setup_CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Rule position is not mandatory anymore, the default is Top
```
Comment (Arsenikr) [inline Playbooks/playbook-PAN-OS_EDL_Setup_CHANGELOG.md]: @yaron-libman

File: Playbooks/playbook-PAN-OS_EDL_Setup.yml (hunk 8 above)
Comment (yaron-libman) [reply Playbooks/playbook-PAN-OS_EDL_Setup.yml]: @Arsenikr I think the sentence should be 'The default position is 'Top'.

**Comment by yaron-libman** [comment]
Review complete.

**Pull request merged**